src/main/java/bio/terra/Main.java

```diff
@@ -21,6 +21,11 @@ public static void main(String[] args) throws Exception {
     // Initially, Jade runs only with Google cloud parts right now, so we set the profile here.
     // ITFOT, we can parameterize the profile to include the appropriate pdao implementations.
     theApp.setAdditionalProfiles("google");
+
+    // Enable sending %2F (e.g. url encoded forward slashes) in the path of a URL which is helpful
+    // if there is a path parameter that contains a value with a slash in it.
+    System.setProperty("org.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH", "true");
```

Comment:
According to a stack overflow user this flag is going away in spring boot 3, which has been out for a while. Maybe this approach would be better, assuming it works in spring boot 2.6/2.7 (and assuming you don't want to update to 3 right now). https://stackoverflow.com/a/76128441

Reply:
Hmm...interesting.  Lemme try that approach.  Could save us a scramble down the road

Reply:
I'm glad you mentioned this.  I am guessing we're a ways away from upgrading to Spring Boot 3 -- even upgrading to from 2.5 -> 2.6 is not working straight out of the gate -- but I could see this biting us in the future.